test/unit/crs.js

```diff
@@ -34,13 +34,21 @@ describe('CRS assertions', function () {
         assert.strictEqual(CRS.toUnit('EPSG:7133'), CRS.UNIT.DEGREE);
         assert.strictEqual(CRS.toUnit('EPSG:4978'), CRS.UNIT.METER);
         assert.strictEqual(CRS.toUnit('EPSG:3857'), CRS.UNIT.METER);
+        assert.strictEqual(CRS.toUnit('EPSG:INVALID'), undefined);
     });
 
     it('should check if the CRS is EPSG:4326', function () {
         assert.ok(CRS.is4326('EPSG:4326'));
         assert.ok(!CRS.is4326('EPSG:3857'));
     });
 
+    it('should assert that the CRS is geocentric', function () {
+        assert.ok(!CRS.isGeocentric('EPSG:4326'));
+        assert.ok(!CRS.isGeocentric('EPSG:7133'));
+        assert.ok(CRS.isGeocentric('EPSG:4978'));
+        assert.ok(!CRS.isGeocentric('EPSG:3857'));
```

Comment:
For unit tests, is it really necessary to test three non geocentric CRS ?

Reply:
(And the same goes for all other tests, there is no benefits in testing the same result multiple times)

Reply:
Don't know, I just tried to have a test symmetric to the `isGeometric` one. I think we should test all four since two here are defined by the test (i.e. the user), one defined by `iTowns` in `Crs` and one by default by `proj4`.